README.md

```diff
@@ -190,19 +189,15 @@ If there are issues, you can enable the 'Enable system diagnostics' check box wh
             - bash: infracost breakdown --path=$(TF_ROOT)/plan.json --format=json --out-file=/tmp/infracost.json
               displayName: Run Infracost
 
-            # See https://github.com/infracost/infracost-azure-devops#infracostcomment for other options
-            - task: InfracostComment@0
-              displayName: Post Infracost comment
-              inputs:
-                githubToken: $(githubToken) # Required to post comments, created in the next step
-                path: /tmp/infracost.json
-                # Choose the commenting behavior, 'update' is a good default:
-                behavior: update # Create a single comment and update it. The "quietest" option.
-                # behavior: delete-and-new # Delete previous comments and create a new one.
-                # behavior: hide-and-new # Minimize previous comments and create a new one.
-                # behavior: new # Create a new cost estimate comment on every push.
-                # Limit the object that should be commented on, either merge-request or commit
-                # targetType: pull-request
+          # Adds a cost estimate comment to a GitHub pull request.
```

Comment:
To have the same style as other comments:
```suggestion
          # Add a cost estimate comment to a GitHub pull request.
```

Also, only line 188 we probably don't need the `Combine Infracost JSON files` comment, as `infracost comment` can do it automatically.

Reply:
done in https://github.com/infracost/infracost-azure-devops/pull/45/commits/51e7994d795d728acc2fcd1bef0007f8f0344df2